components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/main/resources/org.wso2.carbon.identity.configuration.management.yaml

```diff
@@ -408,16 +408,16 @@ paths:
       x-wso2-response: |
         HTTP/1.1 200 Ok
       summary: |
-        Revoke the resource
+        Revoke resource
       parameters:
         - name: resource-name
           in: path
-          description:  This represents the name of the resource to be revoked.
+          description:  This represents the name of the resource that is to be revoked.
           required: true
           type: 'string'
         - name: resource-type
           in: path
-          description:  This represents the type of the resource to be added and can either be the name or id.
+          description:  This represents the type of resource that is to be added and can either be the resource-type name or ID.
```

Comment:
This value can only be the name, not the id. Shall we change it accordingly?

Reply:
Fixed with [c2af0a2](https://github.com/wso2/carbon-identity-framework/commit/c2af0a29f9313baf5105a4898a9e26baceae00fa)